packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js

```diff
@@ -0,0 +1,361 @@
+import {LightningElement} from 'lwc';
+
+import more from '@salesforce/label/c.quantic_More';
+
+/**
+ *  The `QuanticTabBar` component displays the Quantic Tabs in a responsive manner. When tabs are wider than the available space, the tabs that cannot fit in the space are moved in the "More" drop-down list.
+ * @category Search
+ * @example
+ * <c-quantic-tab-bar>
+ *   <c-quantic-tab engine-id={engineId} label="Tab 1" expression={expressionOne} is-active></c-quantic-tab>
+ *   <c-quantic-tab engine-id={engineId} label="Tab 2" expression={expressionTwo}></c-quantic-tab>
+ *   <c-quantic-tab engine-id={engineId} label="Tab 3" expression={expressionThree}></c-quantic-tab>
+ * </quantic-tab-bar>
+ */
+export default class QuanticTabBar extends LightningElement {
+  labels = {
+    more,
+  };
+
+  /** @type {boolean} */
+  hasRendered = false;
+  /** @type {boolean} */
+  isComboboxOpen = false;
+  /** @type {Array<{value: string, label: string}>} */
+  options = [];
+  /** @type {number} */
+  maxMoreButtonWidth = 0;
+  /** @type {boolean} */
+  expandedMoreButton = true;
+
+  connectedCallback() {
+    window.addEventListener('resize', this.updateTabsDisplay);
+    this.addEventListener('tab_rendered', this.updateTabsDisplay);
+  }
+
+  renderedCallback() {
+    if (!this.hasRendered) {
+      this.hideMoreButton();
```

Comment:
To avoid this kind of behaviour when first loading the component:



https://user-images.githubusercontent.com/86681870/169327364-a6a3001d-c4f0-4284-93ad-52447ed1f478.mov